Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-current-contest.tsx

```diff
@@ -282,11 +280,16 @@ const CurrentContestsProvider = ({ children }: ICurrentContestsProviderProps) =>
             return;
         }
 
+        if (!isNil(registerContestError)) {
+            const errorData = registerForContestData as unknown as IErrorDataType;
```

Comment:
Can this be removed 'as unknown as '

Reply:
I need to cast it to uknown before being able to cast it to IErrorDataType.